src/components/Facility/CentralNursingStation.tsx

```diff
@@ -41,6 +40,12 @@ interface Props {
 }
 
 export default function CentralNursingStation({ facilityId }: Props) {
+  const breakpointValues = {
```

Comment:
use usebreakpointhook instead of useeffect and usestate default will render 6 monitor and in big screen it will render 24